verification/src/tests/block_verifier.rs

```diff
@@ -127,22 +127,6 @@ pub fn test_cellbase_with_fee() {
     assert!(verifier.verify(&block).is_ok());
 }
 
-#[test]
-pub fn test_cellbase_overflow_capacity() {
-    let cellbase = TransactionBuilder::default()
-        .input(CellInput::new_cellbase_input(0))
-        .output(CellOutput::new(
-            capacity_bytes!(5),
-            vec![1, 2, 3, 4, 5, 6, 7, 8, 9].into(),
-            Script::default(),
-            None,
-        ))
-        .build();
-    let block = BlockBuilder::default().transaction(cellbase).build();
-    let verifier = CellbaseVerifier::new();
```

Comment:
We can keep the test by using `TransactionsVerifier` here.

Reply:
@doitian it's tested in
https://github.com/nervosnetwork/ckb/blob/584952d42e8d13e21cb15bc1eb166fa11224c7e6/verification/src/tests/transaction_verifier.rs#L21